doc/en/api/1.0.0/featuretypes.yaml

```diff
@@ -4,6 +4,11 @@ info:
   version: 1.0.0
   title: GeoServer Feature Types
   description: A feature type is a vector based spatial resource or data set that originates from a data store. In some cases, such as with a shapefile, a feature type has a one-to-one relationship with its data store. In other cases, such as PostGIS, the relationship of feature type to data store is many-to-one, feature types corresponding to a table in the database.
+  contact:
+    name: GeoServer
+    email: 'geoserver-users@sourceforge.net'
+    url: 'http://geoserver.org/comm/'
+host: localhost:8080
 host: localhost:8080
```

Comment:
`host` is duplicated here, this has broken the hosted rest docs: https://docs.geoserver.org/latest/en/api/#1.0.0/featuretypes.yaml

> Parser error
> duplicated mapping key

Reply:
Everything _should_ be fixed here: https://github.com/geoserver/geoserver/pull/4129